app/policies/application_policy.rb

```diff
@@ -42,6 +42,10 @@ def admin?
     user.admin?
   end
 
+  def inactive?
+    user.inactive?
```

Comment:
This is the check to see if the logged in `user` is allowed to inactivate the `record` they're looking at. The record is a User, which makes this confusing ... but I think you want something like `admin? || country_pcmo?` here.